examples/demo.css

```diff
@@ -196,13 +196,16 @@ h1 {
   overflow: auto;
 }
 
+.no-border {
+  border: 0px;
+}
 
 .savesequencemsg{
   display: none;
   text-align: center;
   }
 
-#stepRemovedNotification {
+#offset-notification,#remove-notification {
```

Comment:
Could we make a generic reusable `.notification` class here? 

We can still leave the `id`s for hiding/showing uniquely. But to have a generic style seems nice and then it can be reused for more things later.

Reply:
Done :+1: